### PR TITLE
[RHPAM-2729] Provide a way to set the logging level on smartrouter image

### PR DESCRIPTION
**Pull request by desmax74**
https://issues.redhat.com/browse/RHPAM-2729

Provide a way to set the logging level on smartrouter image

@jakubschwan @spolti

### CODE REVIEW

File: smartrouter/image.yaml [hunk 1]
```diff
@@ -74,6 +74,12 @@ envs:
     - name: "KIE_SERVER_CONTROLLER_TOKEN"
       example: "dasd373egds"
       description: "KIE server controller token for bearer authentication (Sets the org.kie.server.controller.token system property)"
+    - name: "LOG_LEVEL"
+      example: "INFO"
```
Comment (spolti) [inline smartrouter/image.yaml:78]: change the example to DEBUG and in the description add note that if empty it defaults to info

File: smartrouter/image.yaml [hunk 2]
```diff
@@ -74,6 +74,12 @@ envs:
...
+      description: "Java log level to apply in the logger configuration"
+    - name: "PACKAGES_LOG_LEVEL"
+      example: "PACKAGES_LOG_LEVEL='org.kie=DEBUG \norg.apache=INFO \norg.eclipse=INFO' "
+      description: "fqn with log level, one row for each entry"
```
Comment (spolti) [inline smartrouter/image.yaml:82]: translate fqn to fully qualified name.

File: smartrouter/image.yaml [hunk 3]
```diff
@@ -74,6 +74,12 @@ envs:
     - name: "KIE_SERVER_CONTROLLER_TOKEN"
       example: "dasd373egds"
       description: "KIE server controller token for bearer authentication (Sets the org.kie.server.controller.token system property)"
+    - name: "LOG_LEVEL"
```
Comment (spolti) [inline smartrouter/image.yaml:77]: also, please add some behave tests to test if the file is placed in the correct place.
if LOG_LEVEL is set to DEBUG, also set for debug messages in the smartrouter logs.

File: smartrouter/image.yaml (hunk 1 above)
Comment (desmax74) [reply smartrouter/image.yaml:78]: ok

File: smartrouter/image.yaml (hunk 3 above)
Comment (desmax74) [reply smartrouter/image.yaml:77]: ok

File: smartrouter/image.yaml (hunk 2 above)
Comment (desmax74) [reply smartrouter/image.yaml:82]: ok

File: smartrouter/image.yaml [hunk 4]
```diff
@@ -74,6 +74,12 @@ envs:
...
+      example: "DEBUG"
+      description: "Java log level to apply in the logger configuration, if empty the level used will be INFO"
+    - name: "PACKAGES_LOG_LEVEL"
+      example: "PACKAGES_LOG_LEVEL='org.kie=DEBUG,org.apache=INFO,org.eclipse=INFO'"
+      description: "fully qualified name with log level, in multiple separated by comma"
```
Comment (spolti) [inline smartrouter/image.yaml:82]: Fully qualified logger name

File: smartrouter/image.yaml (hunk 3 above)
Comment (spolti) [reply smartrouter/image.yaml:77]: still not seeing the behave tests on jboss-kie-modules.

Comment (desmax74) [reply smartrouter/image.yaml:77]: adding

Comment (desmax74) [reply smartrouter/image.yaml:77]: @spolti behave test added

File: smartrouter/image.yaml (hunk 4 above)
Comment (desmax74) [reply smartrouter/image.yaml]: Fixed

File: smartrouter/image.yaml [hunk 5]
```diff
@@ -74,6 +74,12 @@ envs:
     - name: "KIE_SERVER_CONTROLLER_TOKEN"
       example: "dasd373egds"
       description: "KIE server controller token for bearer authentication (Sets the org.kie.server.controller.token system property)"
+    - name: "LOG_LEVEL"
+      example: "DEBUG"
+      description: "Java log level to apply in the logger configuration, if empty the level used will be INFO"
+    - name: "PACKAGES_LOG_LEVEL"
```
Comment (spolti) [inline smartrouter/image.yaml:80]: instead use it let's use LOGGER_CATEGORIES to align with other images.

**Comment by spolti** [comment]
will be merged after https://github.com/jboss-container-images/jboss-kie-modules/pull/373

**Pull request merged**